keras_nlp/models/whisper/whisper_decoder.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Whisper decoder block."""
+
+
+from keras_nlp.layers.transformer_decoder import TransformerDecoder
+
+
+class WhisperDecoder(TransformerDecoder):
```

Comment:
I would register these too. I don't think it will affect anything (these are never saved individually), but good practice.